112-Rational-Numbers/rational-1

```diff
@@ -0,0 +1,36 @@
+import (
```

Comment:
mv `112-Rational-Numbers/rational-1` `112-Rational-Numbers/rational-1.gop` // should be a .gop file

Reply:
and to check results by yourself, please refer https://github.com/goplus/tutorial#how-to-run-locally